packages/wonder-blocks-modal/building-blocks.md

```diff
@@ -1 +1,134 @@
 Use these low-level building blocks to build your own modal dialog, instead of using OnePaneDialog. This should happen very rarely and only when a specific exception is required.
+
```

Comment:
@kevinbarabash Turns out adding a `TwoPaneDialog` example was quite easy to build!.

Also I have to mention that `MediaLayout` is 🔥⚡️, it helped me to handle the logic to show the close button in different panels for different screen resolutions (using `mediaSize`).